components/tenant-mgt/org.wso2.carbon.tenant.dispatcher/src/main/java/org/wso2/carbon/tenant/dispatcher/MultitenantDispatcher.java

```diff
@@ -47,16 +48,22 @@ public AxisService findService(MessageContext mc) throws AxisFault {
             String to = mc.getTo().getAddress();
 
             if (to.startsWith(mc.getConfigurationContext().getServiceContextPath())) {
-                int tenantDelimiterIndex = to.indexOf("/t/");
+                int tenantDelimiterIndex = to.indexOf(TENANT_PATH);
                 if (tenantDelimiterIndex != -1) {
-                    AxisConfiguration ac = mc.getConfigurationContext().getAxisConfiguration();
-                    return ac.getService(MultitenantConstants.MULTITENANT_DISPATCHER_SERVICE);
+                    getService(mc);
                 }
+            } else if (to.startsWith(TENANT_PATH)) {
+                getService(mc);
```

Comment:
You should either access static method like `MultitenantDispatcher.getService()` or make `getService() `method non static.

Reply:
Since this fix is reverted, this line of code is not there anymore.